docs/pguidance/agent-install-config-mgmt.md

```diff
@@ -0,0 +1,436 @@
+# Install using configuration management
+
+Install the SignalFx Smart Agent using one of the following configuration management
+packages:
+
+
+
+| Package | See this section                                                                                |
+|:--------|:------------------------------------------------------------------------------------------------|
+| Puppet  | [Install the Smart Agent using a Puppet module](#install-the-smart-agent-using-a-puppet-module) |
```

Comment:
nit: make alphabetical

Reply:
Every command comes from existing documentation. I've tested all the links.